enterprise/frontend/src/embedding-sdk-bundle/store/reducer.ts

```diff
@@ -61,9 +61,7 @@ export const getOrRefreshSession = createAsyncThunk(
     const state = getSessionTokenState(getState() as SdkStoreState);
     const token = storedAuthToken ?? state?.token;
 
-    const isTokenValid = token && token.exp * 1000 >= Date.now();
-
-    if (isTokenValid) {
+    if (!shouldRefreshToken(token)) {
```

Comment:
Make it more clear what we're trying to do here. At first I was reusing a part of `validateSessionToken` function, but in the end we're not doing that.

Here, the condition determines whether we _need_ to request a token, doesn't matter if it's a to issue a token, or to get a new one. We don't actually have a way to _refresh_ token per se. We're re-issuing a new token when the old token expires.